x-pack/solutions/security/plugins/security_solution/server/fleet_integration/fleet_integration.ts

```diff
@@ -258,11 +270,15 @@ export const getPackagePolicyUpdateCallback = (
 
     validateEndpointPackagePolicy(endpointIntegrationData.inputs);
 
-    await notifyProtectionFeatureUsage(
-      endpointIntegrationData,
-      featureUsageService,
-      endpointMetadataService
-    );
+    if (endpointIntegrationData.id) {
+      await notifyProtectionFeatureUsage(
+        endpointIntegrationData,
+        (await endpointServices
```

Comment:
Q: Should we also do some error handling here?

Reply:
Not sure what we would get out of it. Do you mean handle the error in which way? to not throw back to the caller and just "skip" (maybe log a warning) this `notifyProtectionFeatureUsage()` step?

Reply:
Just wondering if `endpointServices` fails for some reason then we get enough info to debug it.

Reply:
Yes that sounds good 👍🏼